database/databasePostgresql/schema.sql

```diff
@@ -4,8 +4,7 @@ CREATE DATABASE calendarteaa;
 \c calendarteaa;
```

Comment:
You would probably want to add your CREATE INDEX statements at the bottom of the file, so it does it automatically after inserting data

Reply:
That is a good idea. I will add that